packages/compass-user-data/package.json

```diff
@@ -50,7 +50,7 @@
     "@mongodb-js/compass-logging": "^1.8.0",
     "@mongodb-js/compass-utils": "^0.9.24",
     "write-file-atomic": "^5.0.1",
-    "zod": "^3.25.17"
+    "zod": "^3.25.76"
```

Comment:
Trying to pin zod versions everywhere to be the same but the problem seems to go deeper than that.